src/ui/component/field/baseField.tsx

```diff
@@ -7,7 +7,7 @@ import './field.scss'
 export type BaseFieldProps = {
   id: string
   inputElement: ReactNode
-  value?: string
+  value?: string | number
```

Comment:
Why not use a bounded generic type? It feels more intuitive for specialization and inheritance to me. Just asking. You tell me.